plugins/obs-filters/obs-filters.c

```diff
@@ -5,42 +5,68 @@ OBS_DECLARE_MODULE()
 
 OBS_MODULE_USE_DEFAULT_LOCALE("obs-filters", "en-US")
 
-extern struct obs_source_info mask_filter;
+extern struct obs_source_info async_delay_filter;
+extern struct obs_source_info chroma_key_filter;
+extern struct obs_source_info color_filter;
+extern struct obs_source_info color_grade_filter;
+extern struct obs_source_info color_key_filter;
+extern struct obs_source_info compressor_filter;
 extern struct obs_source_info crop_filter;
+extern struct obs_source_info crosshatching_filter;
+extern struct obs_source_info cross_stitching_filter;
+extern struct obs_source_info compressor_filter;
+extern struct obs_source_info droste_filter;
 extern struct obs_source_info gain_filter;
-extern struct obs_source_info color_filter;
-extern struct obs_source_info scale_filter;
-extern struct obs_source_info scroll_filter;
 extern struct obs_source_info gpu_delay_filter;
-extern struct obs_source_info color_key_filter;
-extern struct obs_source_info color_grade_filter;
-extern struct obs_source_info sharpness_filter;
-extern struct obs_source_info chroma_key_filter;
-extern struct obs_source_info async_delay_filter;
+extern struct obs_source_info like_a_dream_filter;
+extern struct obs_source_info luma_key_filter;
+extern struct obs_source_info mask_filter;
+extern struct obs_source_info night_vision_filter;
+extern struct obs_source_info noise_gate_filter;
+
 #if SPEEXDSP_ENABLED
-extern struct obs_source_info noise_suppress_filter;
+	extern struct obs_source_info noise_suppress_filter;
```

Comment:
The surrounding content is an if statement, but it's more of an #ifdef then a proper if...
I'll have to a pin in it (like the other one) and get back to you on that.